attributes/default.rb

```diff
@@ -90,18 +91,26 @@
 default['splunk']['splunk_servers'] = []
 
 default['splunk']['forwarder'] = {
-  'url' => value_for_platform_family(
-    %w(rhel fedora suse amazon) => 'https://download.splunk.com/products/universalforwarder/releases/6.6.0/linux/splunkforwarder-6.6.0-1c4f3bbe1aea-linux-2.6-x86_64.rpm',
-    ['debian'] => 'https://download.splunk.com/products/universalforwarder/releases/6.6.0/linux/splunkforwarder-6.6.0-1c4f3bbe1aea-linux-2.6-amd64.deb'
+  'url' => value_for_platform(
```

Comment:
You probably just want to use value_for_platform_family here which would catch the derivatives and wouldn't require the 'default' hash key.

Reply:
Correct me if I'm wrong, Chef doesn't publish what derivatives fall under what platform family, but it is documented what each platform 'String' should be.